extension/manifest.json

```
@@ -1,17 +1,23 @@
     "activeTab",
     "tabs",
     "tabCapture",
-    "offscreen"
+    "offscreen",
+    "storage",
+    "notifications"
   ],
   "host_permissions": [
     "https://music.youtube.com/*"
   ],
+  "icons": {
```

Biome (2.1.2)

[error] The key icons was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)